scripts/create-release-branch.js

```diff
@@ -86,15 +86,21 @@ shell.exec(
 // Add all package.json version update changes
 shell.exec(`git add "**/package.json"`);
 
+// Execute an npm install so that we update the package-lock.json file with the new version 
+// found in the packages for each submodule.
+shell.exec(`npm install --ignore-scripts --package-lock-only --no-audit`);
```

Comment:
So the issue previously is the lerna command to bump the versions doesn't bump them in the package-lock.json file so the packages that had a version update weren't getting installed correctly after running this script.